.github/actions/push-to-artifact-repo/action.yml

```diff
@@ -0,0 +1,22 @@
+name: Push to Artifact Repo
+description: Call push-to-artifact-repo action.
+inputs:
+  source:
+    description: Path to copied source.
+    required: true
+  destination_repo:
+    description: Destination GitHub org/user and repository name (e.g. mongodb/some-artifact).
+    required: true
+runs:
+  using: composite
+  steps:
+    - uses: cbush/push-to-artifact-repo
+      with:
+        email: chris.bush+arty-fact@mongodb.com
+        name: Arty Fact
+        branch: main
+        source: ${{ inputs.source }}
+        toRepo: https://${{ secrets.ARTIFACT_REPO_ACCESS_TOKEN }}@github.com/${{ inputs.target_repo }}
+        commitMessage: Automatic update
+
+          Initiated by ${{ github.server_url }}/${{ github.repository }}/commit/${{ github.sha }}
```

Comment:
i think this is missing `${{ github.repository_owner }}` in the path

Reply:
also would be good if could link to the PR that completes the merge. based on quick googling, it seems a little _awkward_ to access the ref consistently, but [awk](https://www.geeksforgeeks.org/awk-command-unixlinux-examples/) provides a way to get it, per this discussion - <https://github.com/actions/checkout/issues/58#issuecomment-614041550>. 

i think you can have github magically generate link text by including the following string in the message:

```text
${{ github.server_url }}/${{ github.repository_owner}}/${{ github.repository }}/pulls/${{ PR_NUMBER }}
```

and the `PR_NUMBER` you get using the above method with awk in the action before this line.

Reply:
github.repository includes the owner: https://docs.github.com/en/actions/learn-github-actions/contexts

Reply:
The PR# can't be found in the ref after the merge unfortunately:

>For workflows triggered by push, this is the branch or tag ref that was pushed.

Fortunately, the PR# (link and all) is included in the commit message of squash merges anyway.